api/openapi-spec/v3/api__v1_openapi.json

```diff
@@ -5032,6 +5032,10 @@
             "description": "Use the host's pid namespace. Optional: Default to false.",
             "type": "boolean"
           },
+          "hostUsers": {
+            "description": "Use the host's user namespace. Optional: Default to true.",
```

Comment:
```suggestion
            "description": "Use the host user namespace. Optional: Default to true.",
```

Reply:
trying to discern/distill differences between host's user namespaces(hierarchy from host actual root), host user namespace (root/user limited container runtime/shim), and new/further isolated user namespace sub from container runtime/shim.. naming is hard

Maybe if the wording used can be explained somewhere to guide the reader of the code/tests?

Reply:
I am fine with changing it, but we've done it this way to be coherent with what is done for the other namespaces: https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/core/types.go#L3050-L3075

Reply:
@giuseppe +1. I think we want to be consistent with all the other namespaces doc here. We can change the doc for all, in a different PR, if we think that can be improved. But I wouldn't break consistency. It would not be clear if the doc for the user namespaces talks about those things but the doc for the IPC namespace doesn't (a user might think this flag does a different thing to disable to userns and IPC, which is just not true).

Reply:
ok.. I believe the description is wrong as is.  Can you add language like by default when this option is set __... when the option is set to false ____..

Reply:
@mikebrow to mitigate the RTT, can you provide the concrete text you want to see (no blanks to autocomplete)?

I can't think of any way of filling that blanks that is not a NAK for me, as it makes it seems userns is different from others namespaces (like hostIPC, etc.), when it is not.

Reply:
Sure.. here you go using portions of excerpts copied from the KEP and reworded..

```suggestion
            "description": "Use the host's user namespace. Setting hostUsers is Optional: Default is true. If set too true or not present, the pod will be run in the host user namespace, useful for when the pod needs a feature only available to the host user namespace, such as loading a kernel module with CAP_SYS_MODULE. When set too false, a new userns is created for the pod. Setting false is useful for mitigating container breakout vulnerabilities even allowing users to run their containers as root without actually having root privileges on the host",
```

Reply:
@mikebrow thanks, adding it now!

Reply:
@mikebrow added with two minor changes: fixed the typo s/too/to and kept `Optional: Defaults to true` format as that is used in the other fields in this struct (well, the struct were this comes from).

Will be in the next push in a few minutes